tools/depends/target/cmakebuildsys/Makefile

```diff
@@ -18,7 +18,7 @@ BUILD_DIR=$(CMAKE_SOURCE_DIR)/build
 
 all:
 	mkdir -p $(BUILD_DIR)
-	cd $(BUILD_DIR); $(CMAKE) -DCMAKE_BUILD_TYPE=$(Configuration) -DENABLE_INTERNAL_CROSSGUID=OFF -DENABLE_INTERNAL_FFMPEG=OFF $(CMAKE_EXTRA_ARGUMENTS) $(CMAKE_SOURCE_DIR)
+	cd $(BUILD_DIR); $(CMAKE) -DCMAKE_BUILD_TYPE=$(Configuration) -DENABLE_INTERNAL_CROSSGUID=OFF -DENABLE_INTERNAL_FFMPEG=OFF -DHAS_CURL_STATIC=ON $(CMAKE_EXTRA_ARGUMENTS) $(CMAKE_SOURCE_DIR)
```

Comment:
`HAS_CURL_STATIC` is autodetected or did you face issues?

Reply:
it failed without it, saying it couldn't find curl so. after it found curl

Reply:
which platform?

Reply:
android/arm